src/Infrastructure/Persistence/Doctrine/Repository/Regulation/MeasureRepository.php

```diff
@@ -27,4 +27,17 @@ public function delete(Measure $measure): void
     {
         $this->getEntityManager()->remove($measure);
     }
+
+    public function findByRegulationOrderRecordUuid(string $uuid): array
+    {
+        return $this->createQueryBuilder('m')
+            ->where('roc.uuid = :uuid')
+            ->setParameter('uuid', $uuid)
+            ->innerJoin('m.location', 'l')
+            ->innerJoin('l.regulationOrder', 'ro')
+            ->innerJoin('ro.regulationOrderRecord', 'roc')
```

Comment:
(Détail) Dans l'esprit d'une requête SQL où les JOIN sont typiquement écrits avec les WHERE j'aurais écrit ça comme ça

```suggestion
            ->innerJoin('m.location', 'l')
            ->innerJoin('l.regulationOrder', 'ro')
            ->innerJoin('ro.regulationOrderRecord', 'roc')
            ->where('roc.uuid = :uuid')
            ->setParameter('uuid', $uuid)
```